integration_test/weather_transform_impute.dart

```diff
@@ -31,87 +31,79 @@ import 'package:integration_test/integration_test.dart';
 import 'package:rattle/features/impute/panel.dart';
 import 'package:rattle/main.dart' as app;
 
+import 'utils/check_variable_not_missing.dart';
 import 'utils/delays.dart';
+import 'utils/goto_next_page.dart';
 import 'utils/navigate_to_feature.dart';
 import 'utils/navigate_to_tab.dart';
 import 'utils/load_demo_dataset.dart';
 import 'utils/press_first_button.dart';
```

Comment:
Can we please remove `press_first_button.dart`. It should be tapButton(). I have updated tapButton() in dev to find an ElevatedButton with the label rather than simply looking for a widget with the button label and so I think this avoids the problem that tapFirstButton() was attempting to solve.

Reply:
done